content/user/admin_pages/customers/orders.md

```diff
@@ -7,4 +7,21 @@ weight: 20
 
 The Orders page allows you to view your orders. 
 
+It shows your orders in paginated list, with one line per order.  The most recently received order is shown first.  
+
+<img src="/images/orders_list.png" alt="Zen Cart Orders Screen" />
+<br /><br />
```

Comment:
Let's stop using the `/` in `<br />` when adding new HTML content. `<br>` is fine. 
And is the trailing `/` actually needed on `img` tags anymore?